File: VB/TreeListFilter/FilterTreeList/ColumnFilter/PropertyChangedEventArgs.vb
Imports System

Namespace FilterTreeListControl
	Public Class FilterConditionPropertyChangedEventArgs
		Inherits EventArgs
		Private ReadOnly _changedPropertyName As String = ""

		Public Sub New(ByVal propertyName As String)
			Me._changedPropertyName = propertyName
		End Sub

		Public ReadOnly Property ChangedPropertyName() As String
			Get
				Return _changedPropertyName
			End Get
		End Property
	End Class
End Namespace

File: CS/TreeListFilter/FilterTreeList/ColumnFilter/PropertyChangedEventArgs.cs
using System;

namespace FilterTreeListControl
{
	public class FilterConditionPropertyChangedEventArgs : EventArgs
	{
		readonly string changedPropertyName = "";

		public FilterConditionPropertyChangedEventArgs(string propertyName)
		{
			this.changedPropertyName = propertyName;
		}

		public string ChangedPropertyName
		{
			get { return changedPropertyName; }
		}
	}
}
